include/mbedtls/aes.h

```diff
@@ -38,6 +38,7 @@
 
 #define MBEDTLS_ERR_AES_INVALID_KEY_LENGTH                -0x0020  /**< Invalid key length. */
 #define MBEDTLS_ERR_AES_INVALID_INPUT_LENGTH              -0x0022  /**< Invalid data input length. */
+#define MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE               -0x006E  /**< Feature not available, e.g. unsupported AES key size. */
```

Comment:
The problem is that as far as I can tell, are the error codes in the interval [0x0024, 0x002E] are taken. Also, note that according to the doxygen documentation in "error.h" bit 1 is unused, so I do not think I should use it, hence why all the error codes are even. The closest next available error code was 006E.